src/ui/manage-ui/src/FamilyHubs.ServiceDirectory.Admin.Web/Areas/VcsAdmin/Pages/ViewOrganisation.cshtml.cs

```diff
@@ -68,7 +73,16 @@ public async Task<IActionResult> OnPost()
         return outcome.FailureResult!;
     }
 
-    private async Task<Outcome<OrganisationDetailsDto,IActionResult>> SetOrganisationDetails(bool? updated = false)
+    private async Task<IEnumerable<Service>> GetServicesBelongingToOrganisation() =>
```

Comment:
This might be better as a method rather than an inline as its a bit complicated to read

Reply:
Simplified method: https://github.com/DFE-Digital/fh-services/pull/217/commits/1d04f15d7ba465a8116e701ca3711428dec1771b

Also realised that the `ServiceNameDto` object that endpoint returns actually solely contains an Id and a Name, so was no need to create an extra class with the same fields.